agent/config/runtime_test.go

```diff
@@ -6018,6 +6018,7 @@ func TestSanitize(t *testing.T) {
 		"EnterpriseRuntimeConfig": ` + entRuntimeConfigSanitize + `,
 		"ExposeMaxPort": 0,
 		"ExposeMinPort": 0,
+		"EnqueueLimit": "0s",
```

Comment:
Please check other config options, there are more tests in this file for `json` and `hcl` representations.

Reply:
Ah I see. I also updated `TestFullConfig`. Do I have other places to update? Thanks for catching this!